plugins/azure-devops-common/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "@backstage/plugin-azure-devops-common",
```

Comment:
Just wanna get a second opinion here on if this is the name is correct from @backstage/maintainers. Weather it should actually be prefixed with `plugin` even though it's not technically a plugin etc.

Reply:
Yeah I think that's what we've been doing. It's a common prefix, and having them "grouped" together feels sane.

It's almost like it should be `@backstage-plugin/azure-devops-common`, but we never thought of that